packages/db/prisma/schema.prisma

```diff
@@ -132,8 +132,8 @@ model AilaUserFlag {
 }
 
 enum AilaUserFlagType {
-  INAPPROPRIATE_CONTENT
-  INACCURATE_CONTENT
+  INAPPROPRIATE
+  INACCURATE
```

Comment:
@tomwisecodes will this cause an issue with existing flags?

Reply:
Is this your change or mine? if yours will need migrations against prod and staging

Reply:
not mine